pmlc/rt/level_zero/level_zero_wrappers.cc

```diff
@@ -138,8 +138,11 @@ void *level_zero_schedule_write(void *host, void *dev, void *invocation,
 }
 
 void level_zero_wait(uint32_t count, ...) {
+  struct timespec before, after;
   // No submit command created in IR, so call by hand
+  SET_TIMER(before);
```

Comment:
May be we shall move this into level_zero_submit, as we may change the position to call level_zero_submit based on comp dialect changes.

Reply:
Yes, it would be better.  I wonder if there is a better way to get run time of a command list on gpu. Right now this record  the time on host side.